modules/osd-create-cluster-ccs.adoc

```diff
@@ -204,30 +204,59 @@ The *Use a PrivateLink* option cannot be changed after a cluster is created.
 +
 .. If you are installing into an existing VPC and you want to enable an HTTP or HTTPS proxy for your cluster, select *Configure a cluster-wide proxy*.
 endif::osd-on-aws[]
+
 ifdef::osd-on-gcp[]
 . Optional: To install the cluster in an existing GCP Virtual Private Cloud (VPC):
 .. Select *Install into an existing VPC*.
 .. If you are installing into an existing VPC and you want to enable an HTTP or HTTPS proxy for your cluster, select *Configure a cluster-wide proxy*.
 endif::osd-on-gcp[]
-
++
 . Click *Next*.
 
+ifdef::osd-on-gcp[]
+. Optional: To install the cluster into a GCP shared VPC:
++
+[IMPORTANT]
+====
+
+To install a cluster into a shared VPC, you must use {product-title} version 4.13.15 or above. Additionally, the shared VPC administrator must enable a project as a host project in their Google Cloud console. For more information, see link:https://cloud.google.com/vpc/docs/provisioning-shared-vpc#set-up-shared-vpc[Enable a host project].
+====
+
+.. Select *Install into GCP shared VPC*.
+.. Specify the **Host project ID**. If the specified host project ID is incorrect, cluster creation will fail.
```

Comment:
Is "host project ID" initial capped or not? You have it written two different ways here.

Reply:
More on this, since it's bold, I am assuming that it's a field name. Is that correct?  If so, I think the rest is fine, though we may want to mirror how it displays throughout the UI.

We tend to avoid future tense too, so maybe "cluster creation fails."

Reply:
Yes, it is field name. Thanks all! 
![image](https://github.com/openshift/openshift-docs/assets/122639474/471488aa-8b5a-442e-a1bd-22a3b6d97ac3)